### ISSUE TITLE
A way to "hide" findings

### ISSUE THREAD

**Issue by vin-hacks** [caido/caido#1380]
### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help us know what to work on
- Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request

### Description

It would be great to have a way to hide findings so that false positives or already addressed issues don’t take up unnecessary space. This would make it easier to focus on unresolved or new problems, keeping things more organized and improving workflow. Deleting isn’t a solution here since the same findings will just reappear if encountered again.

I was thinking it could be implemented just like the delete option and a toggle button "show hidden" could be at the top of the findings maybe in the filter row.

### Use cases
1. Hiding false positives
When a finding is a false positive, it can be hidden from the list to ensure it doesn’t distract from actual issues.

2. Hiding addressed findings
Findings that have been addressed or resolved could be hidden to keep the list focused on new ones.

Basically you could hide any finding that you don't want to take up space and don't want to see again as new.

**Comment by Corb3nik** [comment]
@Sytten This'll require a `isHidden` on the 
```
input FilterClauseFindingInput {
  reporter: String
}
```

to accurate count / fetch rows in our lazy table.
